inflection/tools/dictionary-parser/src/main/java/org/unicode/wikidata/ClaimsJsonDeserializer.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2025 Unicode Incorporated and others. All rights reserved.
+ */
+package org.unicode.wikidata;
+
+import com.fasterxml.jackson.core.JacksonException;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class ClaimsJsonDeserializer extends JsonDeserializer<Map<String, List<String>>> {
```

Comment:
I would be nice to have a comment on this:
1. It conversion Wikidata into local format?
2. Converts specific (which?) part of wikidata

Reply:
It's converting the JSON format into a more usable Java class for analysis (not local format).

Reply:
I've added some documentation